pkg/controller/logcollector/logcollector_controller.go

```diff
@@ -592,6 +600,14 @@ func (r *ReconcileLogCollector) Reconcile(ctx context.Context, request reconcile
 			},
 			TrustedBundle: trustedBundle,
 		}),
+		rcertificatemanagement.CertificateManagement(&rcertificatemanagement.Config{
```

Comment:
I think the standard pattern is to merge both of these CertificateManagement components into one that has both serviceaccounts and keypair options, instead of using two - otherwise they will both try to write the trusted bundle (which isn't necessarily a problem, but does add additional writes)

Worth a double-check with @rene-dekker though.

Reply:
Yep, same would apply for the CertificateManagementComponent - we'd probably want to conditionally include the serviceaccount and KPO depending on whether EKS log forwarding is enabled.